test/e2e/tests/confirmations/transactions/shared.ts

```diff
@@ -16,186 +16,6 @@ export type TestSuiteArguments = {
   mockedEndpoint?: MockedEndpoint | MockedEndpoint[];
 };
 
-export async function createContractDeploymentTransaction(driver: Driver) {
```

Comment:
we shouldn't use this methods:
- they are not using page objects -> furthermore, using them obfuscate the fact that they are legacy (in the spec you don't see the locators)
- they follow anti-patterns (use harcoded delays, element.click etc)

instead, we are now explicitly using the existing POM classes and methods (or created the needed ones that were not there).

I didn't create flows, for each of this functions, because they are mostly used in 2-3 places in all our code base